### PR TITLE
[stable23] Update test.yml

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         php-versions: [7.3, 7.4, 8.0]
-        nextcloud-versions: ['master']
+        nextcloud-versions: ['stable23']
```
Comment (skjnldsv) [inline .github/workflows/test.yml:10]: You could use `${{ github.base_ref }}` :shrug:

Comment (nickvergessen) [reply .github/workflows/test.yml]: Seems to break it :(

Comment (skjnldsv) [reply .github/workflows/test.yml]: Not in the matrix, if we're only testing against one version, we don't need a matrix

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See https://github.com/nextcloud/recommendations/pull/473#discussion_r788879894

**Comment by skjnldsv** [comment]
But you can also keep `stable23`, we don't have time to improve everything ahah

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**